### PR TITLE
GG-26360 Remove NoLGPL edition

### PR DESCRIPTION
**Pull request by vveider**

### CODE REVIEW

File: modules/geospatial/README.txt [hunk 1]
```diff
@@ -32,17 +32,3 @@ interested in):
     ...
 </project>
 
-
```
Comment (alamar) [inline modules/geospatial/README.txt:35]: I don't think we should remove this clause.

File: modules/hibernate-4.2/README.txt [hunk 2]
```diff
@@ -32,17 +32,3 @@ interested in):
     ...
 </project>
 
-
-LGPL dependencies
```
Comment (alamar) [inline modules/hibernate-4.2/README.txt:36]: and this one

File: modules/hibernate-5.1/README.txt [hunk 3]
```diff
@@ -32,17 +32,3 @@ interested in):
     ...
 </project>
 
-
```
Comment (alamar) [inline modules/hibernate-5.1/README.txt:35]: and this one

File: modules/hibernate-5.3/README.txt [hunk 4]
```diff
@@ -32,17 +32,3 @@ interested in):
     ...
 </project>
 
-
-LGPL dependencies
```
Comment (alamar) [inline modules/hibernate-5.3/README.txt:36]: and this one

File: modules/kubernetes/config/Dockerfile [hunk 5]
```diff
@@ -20,7 +20,7 @@ FROM java:8
 ENV IGNITE_VERSION 2.0.0-SNAPSHOT
 
 # Set IGNITE_HOME variable.
-ENV IGNITE_HOME /opt/ignite/apache-ignite-lgpl-${IGNITE_VERSION}-bin
```
Comment (alamar) [inline modules/kubernetes/config/Dockerfile:23]: let's change it to /opt/gridgiain/gridgain-community-${IGNITE_VERSION}, also below

File: modules/ml/README.txt [hunk 6]
```diff
@@ -8,10 +8,4 @@ GridGain ML Grid module provides machine learning features, along with involved
 Run from project root:
 mvn clean package -DskipTests -pl modules/ml -am
 
-GridGain binary releases cannot include LGPL dependencies. If you would like to activate native BLAS optimizations
```
Comment (alamar) [inline modules/ml/README.txt:11]: Here, I'm not sure.

File: modules/schedule/README.txt [hunk 7]
```diff
@@ -31,17 +31,3 @@ interested in):
     ...
 </project>
 
-
```
Comment (alamar) [inline modules/schedule/README.txt:34]: And this

File: modules/kubernetes/config/Dockerfile (hunk 5 above)
Comment (vveider) [reply modules/kubernetes/config/Dockerfile:23]: Agreed. Done.

File: modules/hibernate-4.2/README.txt (hunk 2 above)
Comment (slukyano) [reply modules/hibernate-4.2/README.txt:36]: On this and other LGPL-related comments.
The text being removed was never true. GridGain of all editions can include LGPL, and always included it in regular editions. Also, we never had any source releases. This text seems to be just a copy-paste from Apache Ignite which was never altered in the fork.

I suggest we proceed with removing LGPL clauses.

**Comment by alamar** [comment]
OK, looks good given the input from @slukyano

**Pull request merged**